docs/tasks/administer-cluster/change-default-storage-class.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Changing the default StorageClass
+---
+
+{% capture overview %}
+This page shows how to change the default Storage Class that is used to
+provision volumes for PersistentVolumeClaims that have no special requirements.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Why change default storage class
+
+Depending on the installation method, your Kubernetes cluster may be deployed with
+an existing StorageClass that is marked as default. This default StorageClass
+is then used to dynamically provision storage for PersistentVolumeClaims
+that do not require any specific storage class. See
+[PersistentVolumeClaim documentation](/docs/user-guide/persistent-volumes/#class-1)
+for details.
+
+This pre-installed default StorageClass may not fit well with your expected work
```

Comment:
"The pre-installed default StorageClass may not fit well with your expected workload; for example, it might provision storage that is too expensive. If this is the case, you can eitehr change the default StorageClass or disable it completely to avoid dynamic provisioning of storage."